packages/api-bzz-browser/src/utils.ts

```diff
@@ -0,0 +1,70 @@
+import { Readable } from 'readable-stream'
+
+// Taken from https://github.com/JCCR/web-streams-node
+// License: https://github.com/jccr/web-streams-node/blob/master/LICENSE
+
+export class NodeReadable extends Readable {
+  private _webStream: ReadableStream
+  private _reader: ReadableStreamDefaultReader<any>
+  private _reading: boolean
+  private _doneReading?: (value?: unknown) => void
+
+  constructor(webStream: ReadableStream, options?: object) {
```

Comment:
`object` isn't a valid type AFAIK, could you add an interface for it please?

Reply:
[`object` is valid type](https://www.typescriptlang.org/docs/handbook/basic-types.html#object). But it is almost on level of `any`, yet I don't think this is a critical part that would require proper Interface. Do you want me still to write Interface for it?

Reply:
I wasn't aware of the `object` type, thanks!
Seems like the `options` interface already exists in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/readable-stream/index.d.ts#L19 so could you use this one please?